frontend/public/components/factory/table.tsx

```diff
@@ -75,6 +76,8 @@ import {
   VolumeSnapshotKind,
 } from '../../module/k8s';
 import { useTableData } from './table-data-hook';
+import '@patternfly/react-virtualized-extension/src/components/Virtualized/examples/VirtualGrid.example.css';
+import '@patternfly/react-virtualized-extension/src/components/Virtualized/examples/WindowScroller.example.css';
```

Comment:
It's odd that we need to import CSS from an example. Are these the right files to include?

Reply:
Received clarification from PF that those were to only support their example page.
So I moved them to their own `_vitrualized-table.scss` file to support `VirtualizedTable`